docs/platforms/javascript/common/migration/v7-to-v8/v8-new-performance-api.mdx

```diff
@@ -252,3 +252,29 @@ Sentry.startSpan((span: Span) => {
   span.setAttribute("attr", 1);
 });
 ```
+
+### Forcing a sampling decision
+
+Previously in v7, you could force a positive or negative sampling decision when calling `startTransaction` by setting the `sampled` option.
+This would effectively override your <PlatformLink to="/configuration/sampling">sampling configuration</PlatformLink> for the specific transaction.
+In v8, the `sampled` option was removed to align with OpenTelemetry. You can still force a decision by defining a
+`tracesSampler` callback in `Sentry.init` and returning `1` or `0` for specific spans:
+
+```JavaScript
+// v7
+Sentry.startTransition({op: 'function.myFunction', sampled: true});
+
+// v8
+// 1. define a tracesSampler
+Sentry.init({
+  tracesSampler: (samplingContext) => {
+    if (samplingContext.op === 'function.myFunction') {
+      return 1;
+    }
+    return 0.1;
```

Comment:
The text above talks about `1` or `0`, but there's a value in-between used here. Is that on purpose? I'd probably rewrite the paragraph above if so.

Reply:
I added an example for a positive (1) and negative (0) decision, as well as a comment to show that the 0.1 is used as a default sample rate